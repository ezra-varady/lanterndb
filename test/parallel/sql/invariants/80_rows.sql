SELECT COUNT(*) FROM small_world;
