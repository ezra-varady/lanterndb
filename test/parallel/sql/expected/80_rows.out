SELECT COUNT(*) FROM small_world;
 count 
-------
    80
(1 row)

